SmartDeviceLinkTests/DevAPISpecs/SDLVoiceCommandManagerSpec.m

```diff
@@ -49,6 +49,9 @@ @interface SDLVoiceCommandManager()
 
     __block SDLVoiceCommand *testVoiceCommand = [[SDLVoiceCommand alloc] initWithVoiceCommands:@[@"Test 1"] handler:^{}];
     __block SDLVoiceCommand *testVoiceCommand2 = [[SDLVoiceCommand alloc] initWithVoiceCommands:@[@"Test 2"] handler:^{}];
+    __block SDLVoiceCommand *testVoiceCommand3 = [[SDLVoiceCommand alloc] initWithVoiceCommands:@[@"Test 3", @" ", @"Test 4"] handler:^{}];
+    __block SDLVoiceCommand *testVoiceCommand4 = [[SDLVoiceCommand alloc] initWithVoiceCommands:@[@""] handler:^{}];
```

Comment:
Can we add another `testVoiceCommand` that has a string with multiple white space characters like so: `@"\t   "` and make sure that it doesn't get sent?